### PR TITLE
Fixes #393 #396 #397 #398 #399 GtfsXXX issues

### LINKED ISSUE #393: GtfsTrip [hasService]
in GtfsTrip datamodel, hasService attribute is optional, when in GTFS trip.txt file its mapped service_id column is mandatory. This attribute SHOULD be marked as mandatory in the datamodels

### PR DESCRIPTION
**Pull request by emastrosimone**
This PR fixes the following GtfsXXX issues:
#393 #396 #397 #398 #399

### CODE REVIEW

**Comment by emastrosimone** [comment]
@jmcanterafonseca  could you review and merge this? 
Thanks

File: specs/UrbanMobility/GtfsRoute/schema.json [hunk 1]
```diff
@@ -41,6 +41,9 @@
         "operatedBy": {
           "type": "string",
           "format": "uri"
+        },
+        "shortName": {
+          "type": "string"
```
Comment (jmcanterafonseca) [inline specs/UrbanMobility/GtfsRoute/schema.json]: isn't this required?

File: specs/UrbanMobility/GtfsStopTime/schema.json [hunk 2]
```diff
@@ -46,6 +46,10 @@
             "0", "1", "2", "3"
           ]
         },
+		"stopHeadsign": {
```
Comment (jmcanterafonseca) [inline specs/UrbanMobility/GtfsStopTime/schema.json]: check indentation and content, this is a string ...

File: specs/UrbanMobility/GtfsRoute/schema.json (hunk 1 above)
Comment (emastrosimone) [reply specs/UrbanMobility/GtfsRoute/schema.json]: The specification [here](https://developers.google.com/transit/gtfs/reference/#routestxt) states:
> At least one of route_short_name or route_long_name must be specified, or potentially both if appropriate. If the route does not have a short name, please specify a route_long_name and use an empty string as the value for this field.

So I guess the fix should be (following the approach already used for POIs in this [PR](https://github.com/FIWARE/dataModels/pull/377)
```
.....
  "allOf": [
    {
      "anyOf": [
        {
          "required": [
            "name"
          ]
        },
        {
          "required": [
            "shortName"
          ]
        }
      ]
    },
	{
      "required": [
        "id",
        "type",
        "routeType",
        "operatedBy"
      ]
    },
......
```

File: specs/UrbanMobility/GtfsStopTime/schema.json (hunk 2 above)
Comment (emastrosimone) [reply specs/UrbanMobility/GtfsStopTime/schema.json]: Fixed in 63357ef

File: specs/UrbanMobility/GtfsRoute/schema.json (hunk 1 above)
Comment (emastrosimone) [reply specs/UrbanMobility/GtfsRoute/schema.json]: Fixed in 3d7dacd

**Comment by jmcanterafonseca** [comment]
build is broken, please check

https://travis-ci.org/FIWARE/dataModels/jobs/514690470

**Comment by emastrosimone** [comment]
> build is broken, please check
> 
> https://travis-ci.org/FIWARE/dataModels/jobs/514690470

Fixed in de63f49

**Pull request merged**